src/aliases/placeholder/parser.go

```diff
@@ -0,0 +1,60 @@
+package placeholder
+
+import (
+	"context"
+	"regexp"
+	"sync"
+
+	aliasent "github.com/consensys/quorum-key-manager/src/aliases/entities"
+)
+
+const aliasParseFormat = `{{(?m)(?P<registry>[a-zA-Z0-9-_+]+):(?P<alias>[a-zA-Z0-9-_+]+)}}$`
+
+var (
+	aliasParseRegexOnce sync.Once
```

Comment:
It's a common way to do in Go for that specific case. The fact that it is a global is mellowed by the fact it is scoped by the package, and mostly it is private.
Why I did that is because the initialization of a regexp pattern is expensive and should be done once.
In general, I'm all against global var, but adding a specific struct to just init seems it's gonna add some complexity.
But in another sense, it does contain business logic.

Reply:
done in cf1d1e1